NEWS.rst

```diff
@@ -3,6 +3,94 @@ http://twistedmatrix.com/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 22.1.0.rc1 (2022-01-26)
+===============================
+
+Features
+--------
+
+- support Python 3.10b4 (#10224)
```

Comment:
@graingert indicated in #twisted that 3.10 proper is supported - should this be clarified?

Reply:
I will update...but please send a PR in which the CI is updated to final 3.10. thanks

ci still runs on 3.10.0-rc.1